Sources/WordPressData/Swift/PublicizeInfo+CoreDataClass.swift

```diff
@@ -8,7 +8,8 @@ import WordPressKit
 /// Furthermore, sites eligible for unlimited sharing will still return a `PublicizeInfo` along with its sharing
 /// limitations, but the numbers should be ignored (at least for now).
 ///
-@objc public class PublicizeInfo: NSManagedObject {
+@objc(PublicizeInfo)
+public class PublicizeInfo: NSManagedObject {
```

Comment:
The following annotation implicitly adds a module name to the Objective-C class name. 

```
@objc open class PublicizeInfo

(lldb) po NSStringFromClass(PublicizeInfo.self)
"WordPressData.PublicizeInfo"
```

This breaks most of our code that works with entities (it uses `classNameWithoutNamespaces`).

The fix is to manually set the global name:

```
@objc(PublicizeInfo)
open class PublicizeInfo
```

Reply:
TIL.